src/kpi_app/app.py

```diff
@@ -155,23 +154,18 @@ def read_event_log_edits():
 )
 
 
-# Define a function to execute SQL queries
-def user_query(sql):
+def query_users(sql):
     cursor = user_db.cursor()
     cursor.execute(sql)
     return cursor.fetchall()
 
 
-# Execute the SQL query to select the desired columns from the "identities" table
```

Comment:
We should avoid comments in general, and only use them when it is stricly necessary. 
This is an example of comment that states the obvious.

In line 199 (`# Delete all authors under baseline`) we have a case of good comment: since it is not very simple there to understand what is going on, the comment is at least helpful. But the really best would be to rewrite the line to make it more readable and not need a comment.